snapcraft.yaml

```diff
@@ -162,12 +113,11 @@ parts:
       - libgcr-ui-3-1
       - libglu1-mesa
       - libgpm2
-      - libhogweed5
-      - libicu66
```

Comment:
some of these packages are already provided by the gnome sdk https://github.com/ubuntu/gnome-sdk/blob/gnome-42-2204-sdk/snapcraft.yaml